### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by lunevav**
Fixed an issue when deleted items stay in array as "empty".
Added .idea directory to .gitignore file.

### CODE REVIEW

File: src/components/Table/Table.js [hunk 1]
```diff
@@ -30,15 +30,19 @@ class Table extends Component {
         const activeRows = this.state.activeRowID;
 
         for (let i = 0; i < array.length; i++) {
-            if (activeRows.includes(array[i])) {
-                delete array[i];
+            if (activeRows.includes(array[i].id)) {
+                console.log('deleted - ', this.props.data[i]);
```
Comment (Kronenberg) [inline src/components/Table/Table.js]: remove all console logs

File: src/components/Table/Table.js [hunk 2]
```diff
@@ -63,7 +67,7 @@ class Table extends Component {
     }
 
     render() {
-        console.log(this.state.activeRowID)
+        console.log("activeRowID=", this.state.activeRowID)
```
Comment (Kronenberg) [inline src/components/Table/Table.js]: remove console

File: src/components/Table/Table.js [hunk 3]
```diff
@@ -30,15 +30,19 @@ class Table extends Component {
         const activeRows = this.state.activeRowID;
 
         for (let i = 0; i < array.length; i++) {
```
Comment (Kronenberg) [inline src/components/Table/Table.js]: use Map() or forEach for this case

**Comment by Kronenberg** [comment]
1) Your pull request has 3 commits.
Before we merge this brunch squash and merge your commits into one.
2) Dev #1 your pull request doesn't mean nothing.

**Comment by lunevav** [comment]
Don't know how to manage all those worthless commits. Merging anyway.

**Comment by Kronenberg** [comment]
Delete brunch after you merge it

**Pull request merged**